appveyor.yml

```diff
@@ -22,7 +22,7 @@ install:
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
 
   # Install the dependencies of the project (skip nibabel for speed)
-  - "conda install --yes --quiet setuptools numpy scipy matplotlib scikit-learn nose mayavi pandas h5py PIL patsy"
+  - "conda install --yes --quiet setuptools numpy scipy matplotlib scikit-learn nose mayavi pandas h5py PIL patsy pyside"
```

Comment:
ok so we should clearly say in the doc and even maybe capture at runtime
what backend we support.

if windows and backend != pyside: raise Error

and also skip the tests ... that would avoid the fedora package issue
reported I suppose...